package.json

```diff
@@ -37,11 +46,24 @@
     "karma-spec-reporter": "0.0.31",
     "karma-webpack": "^2.0.5",
     "moment": "^2.22.2",
-    "mustache": "^2.3.2",
-    "webpack": "^3.12.0"
+    "style-loader": "^0.21.0",
+    "vue-loader": "^15.2.4",
+    "vue-template-compiler": "^2.5.16",
+    "webpack": "^4.8.1",
+    "webpack-bundle-analyzer": "^2.13.1"
   },
   "dependencies": {
+    "axios": "^0.18.0",
     "babel-polyfill": "^6.26.0",
-    "uuid": "^3.3.2"
-  }
+    "lodash": "^4.17.10",
+    "qs": "^6.5.2",
+    "striptags": "^2.2.1",
+    "uuid": "^3.3.2",
+    "vue": "^2.5.16"
```

Comment:
Are you already using axios, lodash, qs, and striptags in the JavaScript? If not, I'd remove them. It's easy to add dependencies when you need them.